packages/components/src/styles/components/layout/grid.scss

```
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// LAYOUT > GRID
+//
+
+.hds-layout-grid {
+  display: grid;
+  // The column gap value is subtracted from the column-min-width to prevent overflow & simplify API for consumers
+
+  // Note: "Unitless 0" <length>s aren’t supported in math functions so we use 0px as a fallback value within calc()
+  // https://drafts.csswg.org/css-values/#calc-type-checking
+  grid-template-columns:
+    repeat(
+      auto-fit,
+      minmax(
+        calc(
+          var(--hds-layout-grid-column-min-width, 0px) - var(--hds-layout-grid-column-gap, 0px)
+        ),
+        1fr
+      )
+    );
+
+  // Note: The gap style is defined here to avoid setting it repeatedly for the gap size variants defined below.
+  // For the gap size variants, we override the value of the gap custom properties to set the desired sizes.
+  // These variables can be used by the consumers as an escape hatch if they need to set non-standard gap values (but adds a bit of friction to it)
+  gap: 
```

```suggestion
  gap:
```
[Nit] trailing space

---

fixed locally. thanks